src/Factory/SessionFactory.php

```diff
@@ -58,7 +58,7 @@ class SessionFactory
 	 */
 	public function createSession(App\Mode $mode, App\BaseURL $baseURL, IConfig $config, Database $dba, ICache $cache, LoggerInterface $logger, Profiler $profiler, array $server = [])
 	{
-		$stamp1  = microtime(true);
+		$profiler->startRecording('parser');
```

Comment:
What is this "parser" category? You replaced a few calls with "rendering" in other files of this same PR.

Reply:
See some lines below. We use the "parser" key for the session stuff. And yes, this is quirky. And since "parser" is used for the cache handling, all the "parser" stuff moved to "rendering".

Reply:
Why not call it "session" then?

Reply:
That would be too easy.